widget/list.go

```diff
@@ -186,6 +187,11 @@ func (l *List) scrollTo(id ListItemID) {
 		l.scroller.Offset.Y = y + lastItemHeight - l.scroller.Size().Height
 	}
 	l.offsetUpdated(l.scroller.Offset)
+	defer func() {
```

Comment:
You will want to call this callback whenever `l.currentFocus` or `l.focused` changes, not here in `scrollTo`, since `scrollTo` is invoked sometimes for things that don't change the focus (like list.ScrollToItem). You will probably also want to add an `OnFocusLost` API, or else call `OnFocused` with -1 as the argument whenever the list loses focus.